packages/execute-driver/lib/plugin.js

```diff
@@ -0,0 +1,134 @@
+import BasePlugin from '@appium/base-plugin';
+import _ from 'lodash';
+import path from 'path';
+import cp from 'child_process';
+import B from 'bluebird';
+
+const FEAT_FLAG = 'execute_driver_script';
+const DEFAULT_SCRIPT_TIMEOUT = 1000 * 60 * 60; // default to 1 hour timeout
```

Comment:
I usually also add units name to duration variable names, e.g. `DEFAULT_SCRIPT_TIMEOUT_MS`